src/@next/Switch/SwitchStyle.tsx

```diff
@@ -1,11 +1,31 @@
 import styled from 'styled-components';
-import { Breakpoints, Spacing } from '..';
+import { Breakpoints, Icon, Spacing } from '..';
 
 import { Blue, Neutral } from '../utilities/colors';
+import { Switch, SwitchProps, SwitchStyleProps } from './Switch';
+
+export const IconStyle = styled(Icon)`
+  height: 16px;
+
+  fill: ${({ fill }) => fill};
```

Comment:
can we use `&[data-with-icon] { fill: }` to style this on SwitchStyle?, That way you don't need to create IconStyle

Reply:
done